### DISCUSSION TITLE
现有项目升级3.5后开启mfsu遇到的问题

### DISCUSSION THREAD

**Discussion by CodingOnStar** [umijs/umi#7142] [Q&A]
现有的项目比较大，之前一直用的webpack4打包，现在想提升下开发效率，就升级了umi3.5，准备试一下mfsu，但本地端QA环境一直是白屏，Network和Console如图。本人新手，对打包方面没有实际经验，所以不太明白这种问题所在，这种状态码307，应该往什么方面考虑去寻找问题和修改，谢谢
![截屏2021-08-09 下午2 31 14](https://user-images.githubusercontent.com/25198901/128668138-63ca3b69-a9f5-464b-a84b-925d38fd8ec5.png)
![截屏2021-08-09 下午2 32 44](https://user-images.githubusercontent.com/25198901/128668159-b7fb27b0-227d-4be3-8af6-c652908d96ff.png)
![截屏2021-08-09 下午2 33 46](https://user-images.githubusercontent.com/25198901/128668167-73326335-b5bd-42f8-a6ca-9396dc4f0fc5.png)

**Comment by leftstick** [comment]
mfsu，理论上说打包出的内容会更大。

我个人的理解，mfsu 应用在本地开发时，可以加速非首次编译、热更新的速度。对调试大有裨益。但是应用到release层面，我觉得帮助并不大。

然后回到你的问题，你现在遇到是一些静态文件被重定向了，这个要看你们qa环境是怎么搞得，是否有nginx之类的反向代理的配置。

我的才测试：要么就是你们qa环境针对这些路由做了特殊配置，要么就是静态资源没放对位置，反向代理的工具当成不存在资源，给了个307。

**Reply by LukeLiou** [reply]
> mfsu，理论上说打包出的内容会更大。
> 
> 我个人的理解，mfsu 应用在本地开发时，可以加速非首次编译、热更新的速度。对调试大有裨益。但是应用到release层面，我觉得帮助并不大。
> 
> 然后回到你的问题，你现在遇到是一些静态文件被重定向了，这个要看你们qa环境是怎么搞得，是否有nginx之类的反向代理的配置。
> 
> 我的才测试：要么就是你们qa环境针对这些路由做了特殊配置，要么就是静态资源没放对位置，反向代理的工具当成不存在资源，给了个307。

请问，下面这个问题有解决方案吗？
使用umi3.5 mfsu方案构建流水线后，访问域名，发现http请求数剧增，网站访问速度大幅下降
目测和config.js中配置有关：`mfsu: { production: { output: '.mfsu-production' } },`
